core/arch/arm/plat-imx/a9_plat_init.S

```diff
@@ -1,31 +1,7 @@
 /* SPDX-License-Identifier: BSD-2-Clause */
 /*
  * Copyright (c) 2014, STMicroelectronics International N.V.
- * All rights reserved.
```

Comment:
On behalf of STMicroelectronics, change approved.
etienne

Reply:
@etienne-lms fine by me. I'd like your `Acked-by:` added to the patch, then.